src/components/ec/cuda/kernel/ec_cuda_reduce_ops.h

```diff
@@ -15,7 +15,6 @@ extern "C" {
 #include "ec_cuda_half_sm52.h"
 #include <cuda_bf16.h>
 #include <cuComplex.h>
-#include <assert.h>
```

Comment:
is this not needed for line 134?

Reply:
the header is already included in ucc_assert.h